src/ServiceControl/Notifications/Email/CustomChecksMailNotification.cs

```diff
@@ -0,0 +1,80 @@
+﻿namespace ServiceControl.Notifications.Email
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Contracts.CustomChecks;
+    using Infrastructure.DomainEvents;
+    using NServiceBus;
+    using ServiceBus.Management.Infrastructure.Settings;
+
+    class CustomChecksMailNotification : IDomainHandler<CustomCheckFailed>, IDomainHandler<CustomCheckSucceeded>
+    {
+        readonly IMessageSession messageSession;
+        readonly string instanceName;
+        string instanceAddress;
+        string[] serviceControlHealthCustomCheckIds = {
+            "Audit Message Ingestion Process",
+            "Audit Message Ingestion",
+            "ServiceControl.Audit database",
+            "Dead Letter Queue",
+            "ServiceControl Primary Instance",
+            "ServiceControl database",
+            "ServiceControl Remotes",
+            "Error Message Ingestion Process",
+            "Audit Message Ingestion",
+            "Error Message Ingestion"
+        };
+
+        public CustomChecksMailNotification(IMessageSession messageSession, Settings settings)
+        {
+            this.messageSession = messageSession;
+
+            instanceName = settings.ServiceName;
+            instanceAddress = settings.ApiUrl;
+
+            if (string.IsNullOrWhiteSpace(settings.NotificationsFilter) == false)
+            {
+                serviceControlHealthCustomCheckIds = settings.NotificationsFilter.Split(new[] { '#' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Could be a static array

Reply:
I've added better parsing so that no character is restricted. '#' is a delimiter but when doubled `##` is treated as non-delimiting `#`. I've added tests to cover that.